website/docs/Installation.md

```diff
@@ -1,226 +1,150 @@
+   ```bash
+   deactivate
+   ```
 
-You can find a list of all supported document types by using `autogen.retrieve_utils.TEXT_FORMATS`.
+### Option b: conda
 
-Example notebooks:
+1. Create and activate a Conda environment:
 
-[Automated Code Generation and Question Answering with Retrieval Augmented Agents](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_RetrieveChat.ipynb)
+   ```bash
+   conda create -n pyautogen python=3.10
+   conda activate pyautogen
+   ```
 
-[Group Chat with Retrieval Augmented Generation (with 5 group member agents and 1 manager agent)](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat_RAG.ipynb)
+2. To exit Conda environment:
 
-[Automated Code Generation and Question Answering with Qdrant based Retrieval Augmented Agents](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_qdrant_RetrieveChat.ipynb)
+   ```bash
+   conda deactivate
+   ```
 
+### Option c: poetry
 
-- #### Teachability
+1. Initialize and activate a Poetry environment:
 
-To use Teachability, please install AutoGen with the [teachable] option.
-```bash
-pip install "pyautogen[teachable]"
-```
+   ```bash
+   poetry init
+   poetry shell
+   poetry add pyautogen
+   ```
 
-Example notebook:  [Chatting with a teachable agent](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_teachability.ipynb)
+2. To exit the Poetry environment:
 
+   ```bash
+   exit
+   ```
 
+## Python Version Compatibility
 
-- #### Large Multimodal Model (LMM) Agents
+AutoGen requires **Python version >= 3.8, < 3.12**. Install it using pip:
 
-We offered Multimodal Conversable Agent and LLaVA Agent. Please install with the [lmm] option to use it.
 ```bash
-pip install "pyautogen[lmm]"
+pip install pyautogen
 ```
 
-Example notebooks:
+Refer to the [Migration guide to v0.2](/autogen/docs/Installation#migration-guide-to-v0.2) for updates and changes.
 
-[LLaVA Agent](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_lmm_llava.ipynb)
-
-
-- #### mathchat
-
-`pyautogen<0.2` offers an experimental agent for math problem solving. Please install with the [mathchat] option to use it.
-```bash
-pip install "pyautogen[mathchat]<0.2"
-```
-
-Example notebooks:
+### Optional Dependencies
 
-[Using MathChat to Solve Math Problems](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_MathChat.ipynb)
+- **Docker**: Highly recommended for code execution. Install with `pip install docker`.
+- **Blendsearch**: For hyperparameter optimization. Install with `pip install "pyautogen[blendsearch]<0.2"`.
+- **Retrievechat**: For retrieval-augmented tasks. Install with `pip install "pyautogen[retrievechat]"`.
+- **Teachability**: For teachable agents. Install with `pip install "pyautogen[teachable]"`.
+- **LMM Agents**: For multimodal conversable agents. Install with `pip install "pyautogen[lmm]"`.
+- **Mathchat**: For math problem-solving agents. Install with `pip install "pyautogen[mathchat]<0.2"`.
```

Comment:
Is it a good idea to remove the notebook examples?

Reply:
removed and added the link back to the MathChat notebook